central/notifiers/generic/generic.go

```diff
@@ -260,10 +260,6 @@ func (g *generic) getPassword() (string, error) {
 		return g.creds, nil
 	}
 
-	if g.GetNotifierSecret() == "" {
```

Comment:
Given that we already have a break condition for unauthenticated notifier above at [ln249](https://github.com/stackrox/stackrox/pull/9000/files#diff-ca64a3e566ba97ba1cd970ff1c4a6e1fc69561ae16891c84273aa2fb26f422a6L249) the NotifierSecret should be non-empty if we come here. If it is empty, the decrypt function will eventually error out if it receives empty NotifierSecret.

Reply:
So no change required, we removed the check and it was moot to begin with.